addons/mail/static/src/env/test_env.js

```diff
@@ -6,20 +6,26 @@
  */
 export function addMessagingToEnv(providedEnv = {}) {
     const env = { ...providedEnv };
+    const legacyEnv = owl.Component.env;
+    const isUiPatched = legacyEnv.browser.innerWidth && legacyEnv.browser.innerWidth !== window.innerWidth;
```

Comment:
not a good condition, maybe make a prep PR so that we no longer need `test_env` (or at least get rid of the browser part?)